src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -78,16 +78,29 @@ export class ExposureNotificationService {
     this.backendInterface = backendInterface;
     this.storage = storage;
     this.secureStorage = secureStorage;
+    // initialize
+    this.updateSystemStatus();
   }
 
   async start(): Promise<void> {
-    this.started = true;
```

Comment:
removing this makes this method not atomically stable. It would be possible to call it twice and it would perform side effects twice. I used this flag specifically to prevent that.

Reply:
With current implementation, `start` is only called once and then gated by `started`. That causes an issue that no sync for last check if user opens the app when EN is disabled. I understand your concern about `atomically stable`.  Let me think if there is other way.